server_admin/topics/events/login.adoc

```diff
@@ -126,6 +126,21 @@ The Logging Event Listener logs events to the `org.keycloak.events` log category
 
 To include debug log events in server logs:
 
+ifeval::["{kc_dist}" == "quarkus"]
+. Change the log level for the `org.keycloak.events` category
+. Change the log level used by the Logging Event listener.
+
+To change the log level used by the Logging Event listener, add the following:
+
+[source,bash]
+----
+bin/kc.[sh|bat] start --spi-events-listener-success-level=info --spi-events-listener-error-level=error
```

Comment:
It should be `--spi-events-listener-jboss-logging-success-level=info`. There is no default provider for the SPI so we need to reference the provider id.